frontend/js/src/common/brainzplayer/SoundcloudPlayer.tsx

```diff
@@ -404,7 +405,9 @@ export default class SoundcloudPlayer
   };
 
   render() {
-    const { show } = this.props;
+    const { show, volume } = this.props;
+    this.soundcloudPlayer?.setVolume((volume ?? 100) / 100);
```

Comment:
This shouldn't be called here but rather most likely in the `onReady` function, or when setting up soundcloudPlayer on line 145